DittoDiskUsage/build.gradle

```diff
@@ -21,7 +21,7 @@ dependencies {
     implementation libs.androidx.compose.ui.ui
     implementation libs.androidx.compose.ui.uiToolingPreview
     implementation libs.androidx.navigation.navigationCompose
-    implementation project(path: ':DittoHealthMetrics')
+    implementation libs.live.ditto.healthmetrics
```

Comment:
Not something we need to change but I'm now wondering whether it's possible to programmatically define where the deps are coming from - so the CI build would always default to the published gradle library dependencies vs project path dependencies.

Reply:
I think now that we're pointing to the gradle version instead of the local project, this should be good enough for now.

Reply:
@zmarkan I like that thought longer term, maybe we log an issue to investigate that further in the future?